V2/tests/unit/use_cases/time_entries_use_case_test.py

```diff
@@ -30,3 +30,16 @@ def test__delete_time_entry_function__uses_the_time_entry_service__to_delete_tim
 
     assert time_entry_service.delete.called
     assert expected_time_entry == deleted_time_entry
+
+
+def test__get_latest_entries_function__uses_the_time_entry_service__to_get_last_entries(
```

Comment:
"function" in the test name is redundant

Reply:
We are going to create a ticket, this or next week to make these changes and standardize the code to avoid these possible errors when writing code, so I do not make the change in this PR